draft-ietf-ace-edhoc-oscore-profile.md

```diff
@@ -582,30 +584,31 @@ If RS fails to find or validate AUTH_CRED_C, then RS MUST respond with an error
 
 ## Access Token in External Authorization Data {#AT-in-EAD}
 
-Instead of uploading the access token to the /authz-info endpoint at RS as described in {{c-rs}}, C MAY include the access token either in EDHOC message\_1 or message\_3 by making use of the External Authorization Data fields (see {{Section 3.8 of I-D.ietf-lake-edhoc}}). The former is shown by the example in {{example-with-optimization}}. In the latter case, the access token is encrypted between C and RS, which provides an alternative for protecting potential sensitive information in the access token.
-
-Editor's note: Shall we remove the EAD_1 case? The case POST /token offers already early abort, and the EAD_3 case is equally efficient but offers better protection of the access token.
+Instead of uploading the access token to the /authz-info endpoint at RS as described in {{c-rs}}, C MAY include the access token in EDHOC message\_3 by making use of the External Authorization Data field EAD_3 (see {{Section 3.8 of I-D.ietf-lake-edhoc}}), see example in {{example-with-optimization}}. In this case, the access token is encrypted between C and RS enabling protection of potential sensitive information.
 
 This document defines the EAD item EAD\_ACCESS\_TOKEN = (ead\_label, ead\_value), where:
 
-* ead\_label is the integer value TBD registered in {{iana-edhoc-ead}}, and
-* ead\_value is a CBOR byte string equal to the value of the "access_token" field of the access token response from AS (see {{as-c}}).
+* ead\_label is the integer value TBD registered in {{iana-edhoc-ead}} and
+* ead\_value is a CBOR map object with label either 0 or 1 and byte string value:
 
-This EAD item, which may be used either in EAD\_1 or EAD\_3, is critical, i.e., it is used only with the negative value of its ead\_label, indicating that the receiving RS must either process the access token or abort the EDHOC session (see {{Section 3.8 of I-D.ietf-lake-edhoc}}).
+   * For label 0, ead\_value = { 0 : access\_token }, the map value is the CBOR byte string equal to the value of the "access\_token" field of the access token response from AS, see {{as-c}}.
+   * For label 1, ead\_value = { 1 : session\_id }, the map value is a CBOR byte string equal to the value of the "session\_id" field of the access token response from AS, see {{token-series}}.
 
-Access tokens are only transported in EAD fields for the first access token of a token series and not for the update of access rights.
+This EAD item, which is used in EAD\_3, is critical, i.e., it is used only with the negative value of its ead\_label, indicating that the receiving RS must either process the access token or abort the EDHOC session (see {{Section 3.8 of I-D.ietf-lake-edhoc}}). An endpoint supporting the profile of ACE defined in this document MUST support this EAD item. When EDHOC is used with this profile, this EAD item MUST be included in EAD_3, see {{m_3}}.
```

Comment:
s/An endpoint supporting/A node supporting

A alternative to "node" is "participant", or "Client or Resource Server". But "endpoint" is used to denote resources such as /token and /authz-info (see Section 1.1).